src/smart-components/group/policy/add-policy/policy-content.js

```diff
@@ -0,0 +1,93 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Form,
+  FormGroup,
+  Stack,
+  StackItem,
+  Text,
+  TextArea,
+  TextContent,
+  TextInput,
+  TextVariants,
+  Title
+} from '@patternfly/react-core';
+import Select from 'react-select';
+
+const PolicyContent = (formValue, onHandleChange, selectedRoles, setSelectedRoles, roles) => {
+
+  const onOptionSelect = (selectedValues) =>
+  { setSelectedRoles(selectedValues); };
+
+  const dropdownItems = roles.map(role => ({ value: role.uuid, label: role.name, id: role.uuid }));
+
+  return (
+    <Fragment>
+      <Form>
+        <Stack gutter="md">
+          <StackItem>
+            <Title size="xl">Create a policy</Title>
+          </StackItem>
+          <StackItem>
+            <TextContent>
+              <Text component={ TextVariants.h5 }>Select role(s)</Text>
+            </TextContent>
+          </StackItem>
+          <StackItem>
+            <TextContent>
+              <Text component={ TextVariants.h6 }>Select at least one role.</Text>
+            </TextContent>
+          </StackItem>
+          <StackItem>
+            <FormGroup
+              label="Policy name"
+              fieldId="policy-name"
+            >
+              <TextInput
+                isRequired
+                type="text"
+                id="policy-name"
+                name="policy-name"
+                aria-describedby="policy-name"
+                value={ formValue.policyName }
+                onChange={ (_, event) => onHandleChange({ policyName: event.currentTarget.value }) }
```

Comment:
First argument should be the actual value, but if you want to use the event i am OK with it (that is what the component should be returning in the first place).